include/RAJA/internal/foldl.hpp

```diff
@@ -48,6 +48,28 @@ struct foldl_impl<Op, Arg1> {
   using Ret = Arg1;
 };
 
+#if defined(__cpp_lib_is_invocable) && (__cpp_lib_is_invocable >= 201703L)
```

Comment:
@trws should we be checking this sort of thing here, following the same pattern as for other language features? https://github.com/LLNL/RAJA/blob/develop/include/RAJA/config.hpp.in#L43

Reply:
Normally I'd say yes, but for something we only need to check in one place, I think it's fine for it to be inline like this.  If it needs to be managed for compiler differences or gets checked elsewhere then we should move it to config.

Reply:
I wouldn't mind seeing it moved to config, it's nice to have a list of all the compiler feature we may or may not use in one place.

Reply:
I'd be happy to revise the PR to define a macro in config if that's the consensus.

Reply:
Yeah, might as well.

Reply:
done.   please let me know if further changes are required.